tests/benchmark/compute/scenario/test_transaction_types.py

```diff
@@ -67,6 +67,14 @@ def get_distinct_receiver_list(
         yield pre.fund_eoa(balance)
 
 
+def get_distinct_unfunded_receiver_list(
+    pre: Alloc,
+) -> Generator[Address, None, None]:
+    """Get a list of distinct unfunded receiver accounts."""
+    while True:
+        yield pre.empty_account()
```

Comment:
Should be checked if this account does not end up in genesis as empty ([EIP-161 "empty" ](https://eips.ethereum.org/EIPS/eip-161))